src/widgets/panel/panel.js

```diff
@@ -112,18 +111,19 @@ export default function panel({
 
     const renderPanel = renderer({
```

Comment:
We can remove the indirection created by the `renderer`. We can call avoid to return a new function from the renderer and provide the full list of argument each time call the `render` function. We do this in widgets because we don't have the control on the `render` call (which can be avoided). Here we do have the control (see below). We can do it in another PR though.

```jsx
const renderer = ({
  containerNode,
  bodyContainerNode,
  cssClasses,
  templateProps,
  options,
  hidden
}) => {
  render(
    <Panel
      cssClasses={cssClasses}
      hidden={hidden}
      templateProps={templateProps}
      data={options}
      bodyElement={bodyContainerNode}
    />,
    containerNode
  );
};
```